format/wasm/instructions.go

```diff
@@ -0,0 +1,1539 @@
+package wasm
+
+import (
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+func decodeBlockType(d *decode.D, name string) {
+	b := d.PeekBytes(1)[0]
+	switch b {
+	case 0x40:
+		d.FieldU8(name, scalar.Sym("ε"))
+	case 0x6f, 0x70, 0x7b, 0x7c, 0x7d, 0x7e, 0x7f:
+		d.FieldU8(name, valtypeToSymMapper)
+	default:
+		d.FieldSScalarFn(name, readSignedLEB128)
+	}
+}
+
+func decodeMemArg(d *decode.D, name string) {
+	d.FieldStruct(name, func(d *decode.D) {
```

Comment:
Can add `//nolint:nolint` if it's ok

Reply:
added `//nolint:unparam` @ https://github.com/wader/fq/pull/383/commits/3fca7cc09a434fd4ba657be69c39934d7be826d0